### DISCUSSION TITLE
Error: ENOENT, No such file or directory

### DISCUSSION THREAD

**Discussion by serranomatias** [nftchef/art-engine#278] [Help + Q&A]
Hello! start to appear this problem after a lot of combinations of nested folders. 

failed to load D:\trabajo\DESURE\NFT\Changos\Generacion Nfts\Programacion\art-engine-nested-folder-structure\layers/Skin/Common-skin#4000/Common-Hair/The Professional#525/Eyes/glasses#2190/Glasses-with-mouth#2030/Round Glasses#250/Nose-roundglasses/NARIZ-4#500/z2,NARIZ-4.png Error: ENOENT, No such file or directory 'D:\trabajo\DESURE\NFT\Changos\Generacion Nfts\Programacion\art-engine-nested-folder-structure\layers/Skin/Common-skin#4000/Common-Hair/The Professional#525/Eyes/glasses#2190/Glasses-with-mouth#2030/Round Glasses#250/Nose-roundglasses/NARIZ-4#500/z2,NARIZ-4.png' 

a few folders works well but a some of them throw this error. 
![Captura1](https://user-images.githubusercontent.com/56377270/165397728-27d97c76-cf48-4b79-b2dc-1793973d0f56.PNG)

**Comment by nftchef** [comment]
These errors are generally caused by typos in the folder/layer names in config. 
If the file in the error path actually exists, there could be another issue that would require a bit more of the config.js and file/folder setup to debug
